modules/kubernetes/control-plane-logs/charts/vector/values.yaml

```diff
@@ -0,0 +1,50 @@
+cloudProvider: ""
+secretName: "msg-queue"
+
+azureConfig:
+  resourceID: ""
+  clientID: ""
+  tenantID: ""
+  keyVaultName: ""
+  hostnameKeyVaultSecretName: "eventhub-hostname"
```

Comment:
These two are not secrets, make the values part of the config instead

Reply:
This is now fixed.